examples/command/portals/kafka/instaclustr/docker/run.sh

```diff
@@ -0,0 +1,116 @@
+    #
+    # The identity will also allowed to create a relay in the project at the address `instaclustr`.
+    instaclustr_operator_ticket=$(ockam project ticket --usage-count 1 --expires-in 10m --relay instaclustr)
+
+    # Create an enrollment ticket to enroll the identity used by an ockam node that will run
+    # adjacent to the Instaclustr client app in application_team's network.
+    #
+    # The identity that enrolls with the generated ticket will be given a cryptographically
+    # attestest project membership credential issue by the membership authority.
+    application_team_consumer_ticket=$(ockam project ticket --usage-count 1 --expires-in 10m --relay '*')
+    application_team_producer_ticket=$(ockam project ticket --usage-count 1 --expires-in 10m --relay '*')
+
+    # Invoke `docker-compose up` in the directory that has instaclustr_operator's configuration.
+    # Pass the above enrollment ticket as an environment variable.
+    #
+    # Read instaclustr_operator/docker-compose.yml to understand the parts that are provisioned
+    # in instaclustr_operator's virtual private network.
+    echo; pushd instaclustr_operator; ENROLLMENT_TICKET="$instaclustr_operator_ticket" BOOTSTRAPSERVER="$bootstrap_server" docker compose up -d; popd
+
+    # Invoke `docker-compose up` in the directory that has application_team's configuration.
+    # Pass the above enrollment ticket as an environment variable.
+    #
+    # Read application_team/docker-compose.yml to understand the parts that are provisioned
+    # in application_team's virtual private network.
+    echo; pushd application_team; PRODUCER_ENROLLMENT_TICKET="$application_team_producer_ticket" CONSUMER_ENROLLMENT_TICKET="$application_team_consumer_ticket" docker compose up; popd
+}
+
+# Cleanup after the example - `./run.sh cleanup`
+# Remove all containers and images pulled or created by docker compose.
+cleanup() {
+    instaclustr_auth
+    ./cluster_manager.sh cleanup
+    pushd instaclustr_operator; docker compose down --rmi all --remove-orphans; popd
+    pushd application_team; docker compose down --rmi all --remove-orphans; popd
+}
+
+instaclustr_auth() {
+    #
+    # Set Instaclustr credentials to create and delete a free trial kafka cluster
+
+    # Prompt the user for Instaclustr username and API key
+    echo -n "Enter Instaclustr username: "
+    read instaclustr_username
+    echo -n "Enter Instaclustr API key: "
+    read -s instaclustr_api_key
+    echo
+
+    # Export the variables as environment variables
+    export INSTACLUSTR_USER_NAME="${instaclustr_username}"
+    export INSTACLUSTR_API_KEY="${instaclustr_api_key}"
+}
+# Check if Ockam Command is already installed and available in path.
+# If it's not, then install it.
+if ! type ockam &>/dev/null; then
+    curl --proto '=https' --tlsv1.2 -sSfL https://install.command.ockam.io | bash
+    source "$HOME/.ockam/env"
+fi
+
+# Check that tools we we need installed.
+for c in docker curl jq; do
```

Comment:
The likelihood that someone will have JQ is too low.

Reply:
If we can't avoid using it - lets use it though docker since we already have that dependency.

Reply:
Using jq to extract the necessary details from the API responses [cluster_manager.sh](https://github.com/build-trust/ockam/pull/7956/files#diff-128852991c06403daaf1d4902ab9214252f8d8722c69422643c96dce8f6c6fb0) after a cluster is created in instaclustr. This script runs at the beginning of the run and no docker image is built at that time. Let me think about this.

Reply:
As suggested, will work on removing `jq` dependency as v2 of this example